src/routes.rs

```diff
@@ -39,6 +40,12 @@ lazy_static::lazy_static! {
     static ref START_TIME: u64 = SystemTime::now().duration_since(UNIX_EPOCH).unwrap().as_millis() as u64;
 }
 
+thread_local! {
```

Comment:
Instead of a static variable, you can use `Extension` as we do it below in the `dispense_tokens` method with `Extension(network_config): Extension<SharedNetworkConfig>`.